independent-projects/tools/base-codestarts/src/main/resources/codestarts/quarkus/tooling/maven-wrapper/base/.mvn/wrapper/..gitignore

```diff
@@ -0,0 +1 @@
+maven-wrapper.jar
```

Comment:
It shouldn't be needed as the script only approach shouldn't download this jar. If we have it around, something is not working as expected.

At least that's my understanding.

Reply:
Ok yes I just tested